src/components/HomeAnimation.vue

```diff
@@ -147,8 +147,8 @@
           <p class="is-title-2 c-container-title">Basic Income for your friends and family.</p>
 
           <div class="c-buttons">
-            <a class="c-navbar-item button is-big is-primary" href="/get-started">Get started</a>
-            <a class="c-navbar-item button is-big is-outlined" href="/sponsor">Sponsor</a>
+            <a class="c-navbar-item button is-big is-primary" href="/groupincome.org/get-started">Get started</a>
+            <a class="c-navbar-item button is-big is-outlined" href="/groupincome.org/sponsor">Sponsor</a>
```

Comment:
The issue with these `/groupincome.org/` links is that when we release the final website on groupincome.org, these `/groupincome.org/` prefixes need to not be here. They are only relevant because of how the Github Pages `okturtles.github.io` domain works.

Do you know know of a way we could update this project so that when we build the site for groupincome.org these links do not use this prefix, but when it's deployed on Github Pages it still works?

Reply:
@taoeffect one way is to use `import.meta.env.BASE_URL` instead of `/groupincome.org/` path prefix as mentioned in [here](https://docs.astro.build/en/reference/configuration-reference/#base), but let me take more time on researching for a possible better option and get back to this thread.